fili-core/src/test/groovy/com/yahoo/bard/webservice/web/ResponseFormatResolverSpec.groovy

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web
+
+import spock.lang.Shared
+import spock.lang.Specification
+
+import javax.ws.rs.container.ContainerRequestContext
+
+class ResponseFormatResolverSpec extends Specification {
+
+    @Shared
+    ResponseFormatResolver resolver =  [ accept: {format, containerRequestContext ->
```

Comment:
This should not be reimplemented here. If we want to test our default implementation of the Resolver (and we do) then we should pull this in out into a proper test rather than inlining it in the binder factory.

As is, this test is useless. It doesn't actually test any code that we're using in production!